src/components/ShareDialog.jsx

```diff
@@ -38,26 +42,39 @@ export default function ShareDialog(props) {
 
   return (
     <Dialog open={open} onClose={onClose}>
-      <DialogTitle className={classes.root}>Share</DialogTitle>
+      <DialogTitle className={classes.root}>SHARE</DialogTitle>
       <DialogContent className={classes.root}>
         <DialogContentText className={classes.root}>
           Currently Sharing: {title}
         </DialogContentText>
-        <EmailShareButton url={link}>
-          <EmailIcon size={55} round />
+        <EmailShareButton url={link} className={classes.shareIcon}>
+          <EmailIcon round />
         </EmailShareButton>
-        <FacebookShareButton url={link}>
-          <FacebookIcon size={55} round />
+        <FacebookShareButton url={link} className={classes.shareIcon}>
+          <FacebookIcon round />
         </FacebookShareButton>
-        <TwitterShareButton url={link}>
-          <TwitterIcon size={55} round />
+        <TwitterShareButton url={link} className={classes.shareIcon}>
+          <TwitterIcon round />
         </TwitterShareButton>
-        <LinkedinShareButton url={link}>
-          <LinkedinIcon size={55} round />
+        <LinkedinShareButton url={link} className={classes.shareIcon}>
+          <LinkedinIcon round />
         </LinkedinShareButton>
-        <RedditShareButton url={link}>
-          <RedditIcon size={55} round />
+        <RedditShareButton url={link} className={classes.shareIcon}>
+          <RedditIcon round />
         </RedditShareButton>
+        <Typography
+          variant="overline"
+          component="h1"
+          className={classes.copyText}>
+          {link}
+        </Typography>
+        <Button
+          color="primary"
+          onClick={() => {
+            navigator.clipboard.writeText(`${link}`);
+          }}>
```

Comment:
You can simplify one-line functions by removing the curly brace:
```jsx
          onClick={() => navigator.clipboard.writeText(`${link}`)}>
```

Reply:
Ok I inlined it